thirdparty/build-thirdparty.sh

```diff
@@ -467,8 +467,8 @@ build_brpc() {
     check_if_source_exist $BRPC_SOURCE
 
     cd $TP_SOURCE_DIR/$BRPC_SOURCE
-    ./config_brpc.sh --headers="$TP_INSTALL_DIR/include /usr/include" --libs="$TP_INSTALL_DIR/bin $TP_INSTALL_DIR/lib /usr/lib" --with-glog
```

Comment:
Why do you remove the glog?
We need glog to print brpc's log.

Reply:
fixed